defaults/main.yml

```diff
@@ -1,13 +1,15 @@
 ---
 # defaults file for ansible-role-certbot
 
-# sectigo_account: "{{ vault_sectigo_account }}"
-# sectigo_meta: "{{ vault_sectigo_meta }}"
-# sectigo_regr: "{{ vault_sectigo_regr }}"
-# sectigo_private_key: "{{ vault_sectigo_private_key }}"
+sectigo_domains:       "{{ ansible_fqdn }}"
+#sectigo_san_domains:  server1.domain.example,server2.domain.example
+#sectigo_emailAddress: contact@org
 
-sectigo_domains: "{{ ansible_fqdn }}"
+#sectigo_eab_kid:      received_from_provider
```

Comment:
Not all people use vault. But if considered beneficial or if desireable to use the existing pattern, this could still be changed to 

`#sectigo_eab_kid: vault_sectigo_eab_kid`